webapp/actions/websocket_actions.jsx

```diff
@@ -241,6 +245,12 @@ function handleUserRemovedEvent(msg) {
     }
 }
 
+function handleUserUpdatedEvent(msg) {
+    if (UserStore.getCurrentId() !== msg.user_id) {
+        AsyncClient.getProfiles();
```

Comment:
I think there's an API to just get a single user so we could use for that.

Also, we might not need the user id check here since the user may be updating themselves on a different tab.